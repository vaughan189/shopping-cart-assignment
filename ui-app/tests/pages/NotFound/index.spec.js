import React from "react";
import { shallow } from "enzyme";
import NotFound from "../../../src/pages/NotFound";

jest.mock("react-router-dom", () => ({
  useLocation: () => ({
    pathname: "",
  }),
}));

describe("NotFound Page", () => {
  it("should render the NotFound Page correctly", () => {
    const component = shallow(<NotFound debug />);
    expect(component).toMatchSnapshot();
  });
});
