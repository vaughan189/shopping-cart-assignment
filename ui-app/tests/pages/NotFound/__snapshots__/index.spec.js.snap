// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Page should render the NotFound Page correctly 1`] = `
<div>
  <h3>
    No match for 
    <code />
  </h3>
</div>
`;
